aips/aip-50.md

```diff
@@ -0,0 +1,72 @@
+
+There is a risk that the pool owner can change the commission rate without the staker knowing. To mitigate this, the maximum commission rate change per lockup period is capped and must be executed at least 7 days before the lockup cycle ends. The change does not take effect until the next lockup period, so the staker can unstake before the change takes effect. Wallets and staking UIs should listen to update commission events to build notifications to alert stakers.
+
+### Goals
+This feature will increase the flexibility of pool owners over network/commission rates. Pool owners can adjust operator commission rates as appropriate, based on market conditions and commercial discussions. As such, this will facilitate staking pools to participate in market dynamics. I.e. the pool can be set at a lower commission to attract more stakers, or raise commission when needed. Operators can call the function themselves if they are also the pool owner, or if not, work with the pool owner they are operating for.
+
+### Out of Scope
+Operators will not be able to change the commission rate. In our existing design setup, operators are designated by pool owners.
+
+### Impact
+* *Delegation pool owners* have permission to change the operator commission rate.
+* *Operators* are impacted by commission rate changes, which will impact their rewards.
+* *Delegators* need to watch over commission rate changes so that they are not overcharged. They may assume that the commission rate is fixed.
+
+## Alternative solutions
+There is no alternative solution because the delegation pool module currently does not allow changing the operator commission rate.
+
+Moreover, we decided against allowing operators to change the commission rate, as it would not make sense in the context of our existing design setup. Right now, Operators are designated by pool owners, so eventually, pool owners control everything.
+
+## Specification
+* The signature of the function to change the commission rate is as follows:
+    ```rust
+    public entry fun update_commission_percentage(
+        owner: &signer,
+        new_commission_percentage: u64
+    )
+    ```
+* Only the pool owner can change the operator commission rate.
+* The maximum commission rate is 100%. The minimum commission rate is 0%.
+* The maximum absolute increase rate per lockup cycle is 10%, which is relative to the current commission rate. For example, if the current commission rate is 9%, the new commission rate cannot be higher than 19% in the next lockup cycle. This is to safeguard delegators from sudden changes in commission rate.
+* There is no maximum decrease rate.
+* The minimum remaining lockup time is 7 days. In other words, the commission rate change can be requested only when at least 7 days are left in the current lockup period.
```

Comment:
This is saying that "the commission rate change can be requested only if 7 or more days are left in the current lockup period."

Reply:
Yes, exactly. I just revised it.